src/style.css

```
@@ -1,3 +1,64 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+  --blue_color: #00acdc;
```

it's better not use color names since this color can be changed and will confuse a little 
```suggestion
  --accent_color: #00acdc;
```